src/Controller/Admin/ConfigController.php

```diff
@@ -18,5 +22,26 @@ class ConfigController extends AdministrationBaseController
     /**
      * {@inheritDoc}
      */
-    protected $properties = ['name', 'context', 'content', 'application_id'];
+    protected $readonly = false;
+
+    /**
+     * {@inheritDoc}
+     */
+    protected $properties = ['name', 'context', 'json' => 'content', 'applications' => 'application_id'];
+
+    /**
+     * {@inheritDoc}
+     */
+    public function beforeFilter(Event $event): ?Response
+    {
+        parent::beforeFilter($event);
+        $response = $this->apiClient->get('/admin/applications', ['filter' => ['enabled' => 1]]);
+        $applications = array_merge(
```

Comment:
`array_merge` don't preserve numeric keys => resulting array can have wrong `id` as key 

```suggestion
        $applications = ['' => __('No application')] + Hash::combine($response['data'], '{n}.id', '{n}.attributes.name');
```